### ISSUE TITLE
#header min-height brakes markup if menu includes count elements more than length of single line

### ISSUE THREAD

**Issue by holdmann** [akabekobeko/redmine-theme-minimalflat2#69]
#header min-height brakes markup if menu includes count elements more than length of single line.

Screenshot - https://yadi.sk/i/pY55J65drpQD5

Tested on mac os x 10.11 Firefox 46.0.1 & Safari 9.0

**Comment by akabekobeko** [comment]
memo:

* End will be hidden it's default theme
* It is in the same state as the [Redmine A1 theme](http://www.redminecrm.com/pages/a1-theme) and minimalflat2
* Proposal to scroll the `#main-menu` on the vertical is not so good, Scroll difficult.

```css
// Main menu
#main-menu {
  position absolute
  bottom 0
  height 2.8em
  width 100%
  background-color color_indigo_light
  font-size .8em

  ul {
    margin 0
    padding 0
    height 100%
    overflow-x: scroll;
  }

  li {
    float left
    list-style-type none
    white-space nowrap
  }
}
```

![scrollable-y](https://cloud.githubusercontent.com/assets/1257526/15462723/97cf8254-20ff-11e6-8cd2-86dfa146cd29.png)

**Comment by akabekobeko** [comment]
memo:

Might this issue is resolved by the new features of Redmine v3.3.

[Patch #20632: Tab left/right buttons for project menu - Redmine](http://www.redmine.org/issues/20632)

**Comment by akabekobeko** [comment]
@holdmann 

Fix a this bug.
Please check with minimalflat2 v1.2.0 and Redmine v3.3.
